src/helm/benchmark/presentation/summarize.py

```diff
@@ -1135,6 +1153,7 @@ def main():
         suite=args.suite, output_path=args.output_path, verbose=args.debug, num_threads=args.num_threads
     )
     summarizer.read_runs()
+    summarizer.get_scenario_spec_instance_ids()
```

Comment:
This name makes it seem like the instance ids are returned.  Call this `read_...` instead.

Reply:
Renamed